.github/workflows/ci.yml

```diff
@@ -49,10 +49,18 @@ jobs:
           ini-values: zend.assertions=1, error_reporting=-1, display_errors=On
           coverage: xdebug
 
-      # Install dependencies and handle caching in one go.
-      # @link https://github.com/marketplace/actions/install-composer-dependencies
+      - name: Determine composer cache directory
+        run: echo "COMPOSER_CACHE_DIR=$(composer config cache-dir)" >> $GITHUB_ENV
+
+      - name: Cache dependencies installed with composer
```

Comment:
Given the frequency at which the CI runs on this repo, I don't think the cache will provide much value. There is 99.9% chances that our dependencies will be updated between our infrequent CI runs.

Reply:
@stof 

Why bother using `ramsey/composer-install` in the first place, then?